app/components/agenda/agendaitem/agendaitem-case/subcase-documents/component.js

```diff
@@ -21,6 +25,9 @@ export default Component.extend(
     init() {
       this._super(...arguments);
       this.set('model', A([]));
+      this.store.query('document-type', { sort: 'priority', size: 50 }).then(types => {
+        this.set('documentTypes', types);
+      });
```

Comment:
Je vraagt hier een size van 50, maar the types in the .then heeft er maar 20:
![image](https://user-images.githubusercontent.com/22245223/81936423-8c7f5200-95f2-11ea-8db1-61e621f249c6.png)

Hierdoor heb ik volgens mij problemen dat ik niet alle types heb?
![image](https://user-images.githubusercontent.com/22245223/81936480-ad47a780-95f2-11ea-9caa-9f26b3866479.png)